.dscanner.ini

```diff
@@ -0,0 +1,67 @@
+; Configurue which static analysis checks are enabled
+[analysis.config.StaticAnalysisConfig]
+; Check variable, class, struct, interface, union, and function names against t
+; he Phobos style guide
```

Comment:
that's from the default config of Dscanner.
I beautified the two cases, were the hard-wrapping didn't work.
Thanks!